lib/linguist/heuristics.rb

```diff
@@ -380,6 +380,22 @@ def call(data)
       end
     end
 
+    disambiguate ".pod" do |data|
+      if /^\s*=\w+$/.match(data)
+        if /^=pod|=cut/.match(data)
+          Language["Pod"]
+        elsif /^\s*=begin pod/.match(data)
```

Comment:
It might be worth testing Perl 6 against `/^\s+=(begin|cut|pod)/` instead (note the use of `\s+`).

To my understanding, Perl 6 allows whitespace to precede Pod lines, whereas Perl 5 doesn't. Testing for leading whitespace would be a good disambiguation.

Reply:
Updated to use your suggested regex.